### PR TITLE
reworked the README and changed the logo

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,56 +12,112 @@
...
 
-### Issues
+### Run your integration
+<p align="left" style="font-size: 18px">
+<img style="width: 30%; height: auto;" src="./images/kaoto-run.png" />
+</p><br/>
 
-Something is not working properly? In that case, feel free to [open issues, add feature requests, report bugs, etc.](https://github.com/KaotoIO/vscode-kaoto/issues).
+### Rapid Prototyping
+[![A video showing how to do rapid prototyping with Kaoto and Camel JBang](https://img.youtube.com/vi/sLuBa5aoWyc/0.jpg)](https://www.youtube.com/watch?v=sLuBa5aoWyc)
```
Comment (apupier) [inline README.md:86]: There is a better integration with VS Code debug adapter than this clipboard thingy which is less convenient and less secure; i think we should not point VS Code users to it.

File: README.md [hunk 2]
```diff
@@ -12,56 +12,112 @@
...
+</p><br/>
 
-### Features
+## Installation
 
-- Edit Camel files following pattern (`*.camel.yaml` and `*.camel.yml`).
-- Edit Kamelet files following pattern (`*.kamelet.yaml` and `*.kamelet.yml`).
-- Edit Pipe files following pattern (`*.pipe.yaml`, `*.pipe.yml`, `*-pipe.yaml` and `*-pipe.yml`)
-- Allow to edit `*.yaml` and `*.yml` when opening through contextual menu.
-- Edit Kaoto (`*.kaoto.yaml` and `*.kaoto.yml`) files. Note that these filename extensions are not supported by Camel JBang and are merely still here to stay backward-compatible with Kaoto v1.
```
Comment (apupier) [inline README.md:37]: the suspported patterns must be visible somewhere as it was source of a lot of questions in the past

File: icon.png
Comment (apupier) [inline icon.png:1]: From where this icon is coming from?

Comment (apupier) [inline icon.png:1]: I think that the image must be max 128pxx128px https://code.visualstudio.com/api/working-with-extensions/publishing-extension#marketplace-integration

**Comment by lhein** [comment]
I reduced that scope of this PR to the bare minimum to get the rebranding finished. The link to the documentation has also been corrected to point to the installation docs.

**Review by apupier — APPROVED** [review]

**Pull request merged**